pages/common/history.md

```diff
@@ -7,7 +7,7 @@
 
 `history`
 
-- Display the last 20 commands:
+- Display the last 20 commands (in `zsh` it displays all commands starting from the 20th):
```

Comment:
```suggestion
- Display the last 20 commands (using `zsh` it displays all commands from history starting from the 20th):
```

PS: I would add either down or up, at the end of that sentence so whichever it is, feel free to edit my suggestion.

Reply:
That's weird, it [deserves a "wat"](https://www.destroyallsoftware.com/talks/wat)! 🙀

I understand that:
* displays all commands from history starting from the 20th using `zsh`
* display the last 20 commands using all other shells

Am I correct?

Reply:
I think you deserve a "wat" for your message, I don't know what you mean!

Reply:
Sorry for my weird comment. :innocent:
By "it deserves a "wat", I meant that it is weird that `history -20` has different behavior depending on the shell (`zsh` / others).
And I wanted to share with you this old conference talk that is very funny! (I think) :performing_arts:

Reply:
> I understand that it:
>
> * displays all commands from history starting from the 20th using zsh
> * displays the last 20 commands using all other shells
>
> Am I correct?

That's how it works on my machine, however it would be nice if someone could verify that / see how other shells behave.

Reply:
for some reason it includes the history command.

Reply:
Generally we've avoided shell-specific examples previously, so if an example is indeed shell specific is there a way we can alter it to make it universal?

Reply:
Doesn't seem to be a way

Reply:
Ah, ok

Let's add this note then, since that  seems to be the best solution to the issue.

@nicokosi Off topic, but that talk was indeed very funny - I remember watching it a few years ago :P

Reply:
What sould we do about @ CleanMachine1's suggestion above?

Reply:
Hrm, I think the existing change in the PR is fine as-is?